tests/Fixtures/AuthorDeprecatedReadOnly.php

```diff
@@ -19,9 +19,10 @@
 class AuthorDeprecatedReadOnly
 {
     /**
-     * @JMS\Serializer\Annotation\ReadOnly
```

Comment:
If I remember correctly the test was setup to avoid issues with not working `ReadOnly` annotation. Is it possible to keep that annotation?

Reply:
This comment fits better on #1494 as that's where the change is coming from (as I said, I based this branch on that one for my own sanity 😆).  But, I was running into issues running the tests locally on PHP 8.2 with the aliased class name.  I can try to put that back and see what happens.